oam/workloads/application.meshery.layer5.io.schema.json

```diff
@@ -17,16 +19,20 @@
         },
         "labels": {
           "type": "object",
-          "additionalProperties": {
-            "format": "string",
-            "type": "string"
-          }    
+          "properties": {
+            "additionalProperties" : {
```

Comment:
can you attach a screenshot of how this changes the way RJSF renders additionalProperties ?

Reply:
![Screenshot from 2022-02-07 20-10-40](https://user-images.githubusercontent.com/65453826/152809416-b2263406-6d40-4eeb-a07e-e23a89715e61.png)